python/cli/prelude_cli/views/build.py

```diff
@@ -36,12 +36,12 @@ def build(ctx):
 def create_test(controller, name, unit, test, technique):
     """ Create or update a security test """
     def create_template(template, name):
-        utc_time = str(datetime.now(timezone.utc))
         template_body = pkg_resources.read_text(templates, template)
-        template_body = template_body.replace('$ID', t['id'])
-        template_body = template_body.replace('$NAME', t['name'])
-        template_body = template_body.replace('$UNIT', t['unit'])
-        template_body = template_body.replace('$TIME', utc_time)
+        template_body = template_body.replace('$ID', t.get('id'))
+        template_body = template_body.replace('$NAME', t.get('name'))
+        template_body = template_body.replace('$UNIT', t.get('unit'))
+        template_body = template_body.replace('$TECHNIQUE', t.get('technique'))
```

Comment:
t.get('technique') will still return None and its the text.replace('blah', None) that errors

i'd go back to the direct `t['blah']` format - the .get()s don't get you anything i dont think

Reply:
I misunderstood, in two different ways. I think we want an empty value if there's no technique, not `None` or `''`.